app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,121 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui yellow inverted segment">
+      <div class="ui tablet stackable grid">
```

Comment:
Instead of a grid, a custom class for the countdown timer ... You could have just used semantic's [statistics](https://semantic-ui.com/views/statistic.html) right ?

```hbs
<div class="ui yellow inverted segment">
    <div class="ui horizontal statistic">
        <div class="value">
            08:48
        </div>
        <div class="label">
            {{t 'Please complete registration within 10:00 minutes.'}}
            <br>
            {{t 'After 10:00 minutes, the reservation we\'re holding will be released to others.'}}
        </div>
    </div>
</div>
```

Looks much cleaner. Less DOM elements and no need of a custom style either.